about.html

```diff
@@ -0,0 +1,78 @@
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>About capstone</title>
+  <link rel="stylesheet" href="about.css">
+</head>
+<body>
+  <nav class="nav-box">
+    <div class="hamburger" onclick="toggleMenu()">
+      <span></span>
+      <span></span>
+      <span></span>
+    </div>
+
+  <ul class="dropdown-menu">
+    <li><a href="index.html">Home</a></li>
+    <li><a href="index.html#services">Services</a></li>
+    <li><a href="index.html#resident-musicians">Resident Musicians</a></li>
+    <li><a href="#about">About</a></li>
+  </ul>
+    <div class="logo">
+        <img src="Images/Logo.png" alt="">
+      </div>
+    <div class="dropdown-desktp">
+      <a href="index.html">Home</a>
+      <a href="index.html#services">Services</a>
+      <a href="index.html#resident-musicians">Resident-Musicians</a>
+       <a href="#about">About</a>
+    </div>
+ </nav>
+
+<section>
+<div class="about-head">
+  <h3>Hello, Creatives Welcome To</h3>
+  <h2>GROOVY STUDIOS.</h2>
+  <div class="about-paDsk">
+  <p>Pride is the lifeblood of our music studio. Our team pours their hearts into each project, maintaining top-quality equipment, fostering inclusivity, and promoting collaboration. We take pride in creating music that moves hearts and minds. Join us and experience the unwavering pride that resonates in every note, beat, and moment of musical magic.</p>
+</div>
+</div>
+<div class="about-para">
+  <p>Pride is the lifeblood of our music studio. Our team pours their hearts into each project, maintaining top-quality equipment, fostering inclusivity, and promoting collaboration. We take pride in creating music that moves hearts and minds. Join us and experience the unwavering pride that resonates in every note, beat, and moment of musical magic.</p>
+</div>
+
+<div class="about-contact">
+  <p>Please contact us per Email for more futher questions about </p>
+  <a href="#">www.groove.co.za</a>
+</div>
+ <div class="Logo-sec">
+  <h2>Groovy studios Logo</h2>
+  <p>The music studio is a creative haven, where melodies and rhythms come alive. Its state-of-the-art equipment, expert engineers, and inspiring atmosphere fuel musical brilliance. A true artist's playground for crafting unforgettable sonic experiences.</p>
+  <hr>
+  <img src="Images/Logo.png" alt="logo">
+ </div>
+ <div class="albums">
+  <h2>Here are some of the albums we worked on.</h2>
+  <h4>Take a listen to two of our most sort after albums, we are very proud to have been the ones who produced,mixed and mastered this two works of art. Enjoy every single moment of magic in this album.a</h4>
+  <div class="album-images">
+    <img src="Images/Album.jpg" alt="Artist">
+    <img src="Images/Album.jpg" alt="Artist">
+  </div>
+  
+  
+ </div>
+</section>
+ <footer>
+  <img src="Images/Logo.png" alt="logo">
+  <h6>
+    Groovy studios.
+    All Rights reserved
+  </h6>
+</footer>
```

Comment:
Great design @God-Logic 🤝

- [ ] Kindly ensure your footer section replicates the design given.
- Your design.
![your-about-footer](https://user-images.githubusercontent.com/63917725/233806203-0356f0c3-6045-4f04-8523-7cfd5472c8c1.PNG)
- Given design
![given-about-footer](https://user-images.githubusercontent.com/63917725/233806217-00e86c37-56ee-4406-81cf-51ede76e2785.PNG)